PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue

```diff
@@ -237,6 +238,19 @@ export default {
         message += " Gender Identity";
         valid = false;
       }
+ 
+      if( this.householdPatientPhoto ) {
```

Comment:
Can you combine these like you did elsewhere?

```
if(this.householdPatientPhoto && this.householdPatientPhoto.size > 2097152) {
...
```

Reply:
I'm not sure where that curvy top of the image is coming from.